src/plugins/embeddable/public/lib/state_transfer/types.ts

```diff
@@ -20,6 +20,8 @@
 import { Optional } from '@kbn/utility-types';
 import { EmbeddableInput, SavedObjectEmbeddableInput } from '..';
 
+export const EMBEDDABLE_EDITOR_STATE_KEY = 'embeddable_editor_state';
```

Comment:
bit of a nit but this is more of a constant rather than a type, right? maybe could go in a constants files if applicable

Reply:
A constants file would be a good home for this, but the embeddable plugin doesn't have one yet.